src/openscad.cc

```diff
@@ -446,6 +446,13 @@ int cmdline(const CommandLine& cmd)
     render_variables.time = 0;
     return do_export(cmd, render_variables, export_format, root_file);
   } else {
+    std::shared_ptr<OffscreenView> glview = nullptr;
+    if ((export_format == FileFormat::ECHO || export_format == FileFormat::PNG) && (cmd.viewOptions.renderer == RenderType::OPENCSG || cmd.viewOptions.renderer == RenderType::THROWNTOGETHER)) {
+      Camera camera = cmd.camera;
```

Comment:
This looks a bit dubious: You're copying the camera, then passing the copy into create_offscreenview() which potentially modifies the camera, then passing the original camera to do_export() later.